run.mac.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+osname=$(uname)
```

Comment:
It seems like if I run a job via the CLI and then try to run another job
inside StochSS, the job gets stuck in Pending in stochss.

If I just run something on a fresh AMI launched in StochSS it's fine.

Ben

On Sun, Feb 8, 2015 at 3:10 PM, Brian Drawert <notifications@github.com>
wrote:

> In run.mac.sh
> <https://github.com/StochSS/stochss/pull/281#discussion_r24303740>:
>
> > @@ -1,5 +1,12 @@
> >  #!/usr/bin/env bash
> >
> > +osname=$(uname)
>
> What is the purpose of the changes to these two files?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/StochSS/stochss/pull/281/files#r24303740>.
>